search-commons/src/main/java/no/unit/nva/search/constants/ApplicationConstants.java

```diff
@@ -63,6 +50,23 @@ public final class ApplicationConstants {
         List.of(TYPE_TERMS_AGGREGATION,
                 STATUS_TERMS_AGGREGATION
         );
+    public static final String ENTITY_DESCRIPTION = "entityDescription";
+    public static final String CONTRIBUTORS = "contributors";
+    public static final String NAME = "name";
+    public static final String  ="name";
+    public static final String IDENTITY = "identity";
+    public static final List<AbstractAggregationBuilder<? extends AbstractAggregationBuilder<?>>>
+        RESOURCES_AGGREGATIONS = List.of(
+
+        generateSimpleAggregation("entityDescription.reference.publicationInstance.type",
+                                  "entityDescription.reference.publicationInstance.type.keyword"),
+        generateSimpleAggregation("resourceOwner.owner",
+                                  "resourceOwner.owner.keyword"),
+        generateSimpleAggregation("resourceOwner.ownerAffiliation",
+                                  "resourceOwner.ownerAffiliation.keyword"),
+        generateContributorAggregation(),
+        generateObjectLabelsAggregation("topLevelOrganization")
```

Comment:
generateObjectLabelsAggregation kan ikke brukes til noe annet enn aggregeringen av topLevelOrganization pga. metoder den kaller. Kanskje best å ikke sende strengen inn i metoden? For det virker litt misvisende, at metoden kan brukes for aggregering av andre verdier.